internal/tiltfile/value/label.go

```diff
@@ -0,0 +1,87 @@
+package value
+
+import (
+	"fmt"
+	"strings"
+
+	"go.starlark.net/starlark"
+	validation "k8s.io/apimachinery/pkg/util/validation"
+)
+
+type LabelValue string
+
+func (lv *LabelValue) Unpack(v starlark.Value) error {
+	str, ok := AsString(v)
+	if !ok {
+		return fmt.Errorf("Value should be convertible to string, but is type %s", v.Type())
+	}
+
+	validationErrors := validation.IsQualifiedName(str)
```

Comment:
should this use IsValidLabelValue?

the way you've defined labels, they need to be both valid label names and valid label values